docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,332 @@
+    save_results(data)
+```
+
+## Step 2: Create a Schedule
+
+Now, let's create a schedule for our pipeline. We'll set it to run daily at 9 AM.
+
+```python
+from zenml.config.schedule import Schedule
+from datetime import datetime
+
+# Create a schedule that runs daily at 9 AM
+schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 9 * * *"  # Run at 9 AM every day
+)
+
+# Attach the schedule to our pipeline
+scheduled_pipeline = daily_data_pipeline.with_options(schedule=schedule)
+
+# Run the pipeline to create the schedule
+scheduled_pipeline()
+```
+
+### Best Practice: Use Descriptive Schedule Names
+
+When creating schedules, follow a consistent naming pattern to better organize them:
+
+```python
+# Example of a well-named schedule
+schedule = Schedule(
+    name="daily-feature-engineering-prod-v1",
+    cron_expression="0 4 * * *"
+)
+```
+
+Include the frequency, purpose, environment, and version in your schedule names.
+
+## Step 3: Verify the Schedule
+
+Let's check if our schedule was created successfully using both Python and the CLI:
+
+```python
+from zenml.client import Client
+
+# Get the client
+client = Client()
+
+# List all schedules
+schedules = client.list_schedules()
+
+# Find our schedule
+our_schedule = next(
+    (s for s in schedules if s.name == "daily-data-processing"),
+    None
+)
+
+if our_schedule:
+    print(f"Schedule '{our_schedule.name}' created successfully!")
+    print(f"Cron expression: {our_schedule.cron_expression}")
+    print(f"Pipeline: {our_schedule.pipeline_name}")
+else:
+    print("Schedule not found!")
+```
+
+Using the CLI to verify:
+
+```bash
+# List all schedules
+zenml pipeline schedule list
+
+# Filter schedules by pipeline name
+zenml pipeline schedule list --pipeline_id my_pipeline_id
+```
+
+## Step 4: Update the Schedule
+
+Sometimes we need to modify an existing schedule. Since ZenML doesn't support direct schedule updates, we'll need to delete the old schedule and create a new one.
+
+```python
+# First, delete the existing schedule
+client.delete_schedule("daily-data-processing")
+
+# Create a new schedule with updated parameters
+new_schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 10 * * *"  # Changed to 10 AM
+)
+
+# Attach the new schedule to our pipeline
+updated_pipeline = daily_data_pipeline.with_options(schedule=new_schedule)
+
+# Run the pipeline to create the new schedule
+updated_pipeline()
+```
+
+Using the CLI to delete a schedule:
+
+```bash
+# Delete a specific schedule
+zenml pipeline schedule delete daily-data-processing
+```
+
+> **Important**: When updating schedules, you should also delete the
+> corresponding schedule in your orchestrator (Vertex AI in this example). You
+> can do this through the Google Cloud Console or using the orchestrator's API
+> (see below for code example).
+> ZenML's delete command may not always completely remove the underlying
+> orchestrator schedule.
+
```

Comment:
I'd also add the underlying GCP snippet to make this more tangible. Maybe borrow from my PR [here ](https://github.com/zenml-io/zenml/blob/42d4040fa96921b136296f81788ef64e039916dd/docs/book/how-to/pipeline-development/build-pipelines/schedule-a-pipeline.md#updatepausestop-a-schedule) and [here](https://github.com/zenml-io/zenml/blob/42d4040fa96921b136296f81788ef64e039916dd/docs/book/component-guide/orchestrators/vertex.md#run-pipelines-on-a-schedule)